src/main/java/mops/foren/infrastructure/web/LogoutController.java

```diff
@@ -0,0 +1,18 @@
+package mops.foren.infrastructure.web;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.context.annotation.SessionScope;
+
+import javax.servlet.http.HttpServletRequest;
+
+@Controller
+@SessionScope
```

Comment:
I think we need to add a request mapping to /foren. 
To be in unity with the dev branch.